scripts/import-new-mdb.sh

```diff
@@ -17,7 +17,7 @@ function import {
   QUERY="SELECT text FROM text_data WHERE category = $3"
 
   sqlite3 "$1" "$QUERY" > "$TEMP_FILE"
-  ./scripts/merge-csv.sh "$TEMP_FILE" "$2" > "$TEMP_FILE_2"
+  ./merge-csv.sh "$TEMP_FILE" "$2" > "$TEMP_FILE_2"
```

Comment:
Let's keep the original; I'd like the scripts to be runnable from the root of the project.